docs/how-to/unified_memory.rst

```diff
@@ -0,0 +1,595 @@
+.. meta::
+  :description: This chapter describes introduces Unified Memory (UM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, unified memory, unified, memory, UM, APU
+
+*******************************************************************************
+Unified Memory
+*******************************************************************************
+
+Concept
+=======
+
+In a conventional architectures, both CPUs a GPUs have a dedicated memory,
```

Comment:
Remove 'a' in 

In **a** conventional architectures

Reply:
It has been removed in the last commit, but Github UI does not show it, because there was a comment on it.
You can see the latest version here:
https://github.com/ROCm/HIP/pull/3494/files#diff-9ffae9738c016086230e9386a7d0aa1b26bd7a8f4845494ba9952cf0de47c1db